src/lib/components/works_for_you/notification.tsx

```diff
@@ -0,0 +1,86 @@
+import * as Relay from 'react-relay'
+import * as React from 'react'
+import * as moment from 'moment'
+import { View, StyleSheet, ViewProperties } from 'react-native'
+
+import SerifText from '../text/serif'
+import Headline from '../text/headline'
+import ArtworksGrid from '../artwork_grids/generic_grid'
+
+import colors from '../../../data/colors'
+
+interface Props extends ViewProperties {
+  notification: {
+    message: string
+    date: string
+    artists: string
+    status: 'READ' | 'UNREAD'
+    artworks: any[]
+  }
+}
+
+class Notification extends React.Component<Props, any> {
+  render() {
+    const notification = this.props.notification
+    const formattedDate = moment().format('MMM D')
+
+    return (
+      <View style={styles.container}>
+        <View style={styles.header}>
+          <View>
+            <Headline style={styles.artistName}>{notification.artists}</Headline>
+            <SerifText style={styles.metadata}>{notification.message + ' · ' + formattedDate}</SerifText>
+          </View>
+          { notification.status === 'UNREAD' && <View style={styles.readStatus}/>}
+        </View>
+        <View style={styles.gridContainer}>
+          <ArtworksGrid artworks={notification.artworks.slice(0,3)}/>
```

Comment:
I sliced the response because staging provides way more artworks than are needed; will update once production is ready.